src/fp12.rs

```diff
@@ -224,7 +315,7 @@ impl Fp12Compressed {
     }
 }
 
-impl crate::traits::Compress for Fp12 {
+impl Compress for Fp12 {
```

Comment:
Probably should implement this for `Gt` as well, as that is the main use case for the compression we have in `bellperson`

Reply:
Done